3-fleetscope/modules/env_baseline/acm.tf

```diff
@@ -85,7 +85,9 @@ resource "google_project_iam_binding" "acm_wi_trace_agent" {
   role = "roles/cloudtrace.agent"
   members = [
     "principal://iam.googleapis.com/projects/${data.google_project.cluster_project.number}/locations/global/workloadIdentityPools/${var.fleet_project_id}.svc.id.goog/subject/ns/config-management-monitoring/sa/default",
-    "principal://iam.googleapis.com/projects/${data.google_project.cluster_project.number}/locations/global/workloadIdentityPools/${var.fleet_project_id}.svc.id.goog/subject/ns/default/sa/cymbal-bank", #TODO rename/move
+    "principal://iam.googleapis.com/projects/${data.google_project.cluster_project.number}/locations/global/workloadIdentityPools/${var.fleet_project_id}.svc.id.goog/subject/ns/accounts-${var.env}/sa/cymbal-bank",
```

Comment:
We shouldn't hardcode these (`accounts`, `ledger`, `frontend`, and `cymbal-bank`) in the baseline.  Can we instead  assemble using the list of `teams` and `apps`?

Reply:
Since we currently don't have a reference for cymbal-bank on this step. I opted out for using principalset: "All Pods in a namespace, regardless of service account or cluster". So all kubernetes SA created on the namespaces that were created in 3-fleetscope will have the baseline roles for monitoring and cloud trace

Reply:
I also added a variable in case the user wants to add any additional identities